include/nbl/builtin/hlsl/binops.hlsl

```diff
@@ -37,6 +37,15 @@ struct bitwise_xor
     }
 };
 
+template<typename T>
+struct bitwise_add
```

Comment:
`bitwise` add XD ?

mul, min?

 wtf!?

Reply:
Don't we need them? Aren't these supposed to be used by reduce and scan ops? Or is it the prefix that's bad?